src/bicep/modules/firewall.bicep

```diff
@@ -27,11 +27,101 @@ resource firewallPolicy 'Microsoft.Network/firewallPolicies@2021-02-01' = {
   }
 }
 
+resource firewallAppRuleCollectionGroup 'Microsoft.Network/firewallPolicies/ruleCollectionGroups@2021-02-01' = {
+  name: '${firewallPolicyName}/DefaultApplicationRuleCollectionGroup'
+  dependsOn: [
+    firewallPolicy
+  ]
+  properties: {
+    priority: 300
+    ruleCollections: [
```

Comment:
I think since this is such a complex object it makes sense to pass in as a `json(loadTextContent('./firewallRules/applicationRules.json')` and document for users how to update this file

Reply:
Agreed, although I would think possible at the 'ruleCollection' level.